app/cafe.py

```diff
@@ -0,0 +1,25 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError, OutdatedVaccineError, NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+
+        if ("vaccine" not in visitor
+                or "expiration_date" not in visitor["vaccine"]):
+            raise NotVaccinatedError
+
+        vac_expire = visitor["vaccine"]["expiration_date"]
+        today = datetime.date.today()
+        if today > vac_expire:
+            raise OutdatedVaccineError
```

Comment:
The `OutdatedVaccineError` should be raised with a descriptive message. According to the checklist, you should provide a message when raising exceptions. Consider using the message from the `__str__` method of `OutdatedVaccineError` in `errors.py`.